src/pkg/packager/creator/normal.go

```diff
@@ -61,7 +61,8 @@ func NewPackageCreator(createOpts types.ZarfCreateOptions, cwd string) *PackageC
 }
 
 // LoadPackageDefinition loads and configures a zarf.yaml file during package create.
-func (pc *PackageCreator) LoadPackageDefinition(ctx context.Context, src *layout.PackagePaths) (pkg types.ZarfPackage, warnings []string, err error) {
+// Use LoadPackageDefinitionWithValidate unless there is a specific reason to skip validation or run validation separately
+func (pc *PackageCreator) LoadPackageDefinition(ctx context.Context, src *layout.PackagePaths) (pkg types.ZarfPackage, warnings []types.PackageError, err error) {
```

Comment:
Should LoadPackageDefinition now just call LoadPackageDefinitionWithValidate, or where (should?) this still be used

Reply:
LoadPackageDefinition should be used where we don't want to validate on a package yet, because we are going to change more in it, such as it's use in `zarf dev deploy` or when we want to do additional validations on the package even if it's invalid, such as in `zarf dev lint`. 

In the regular case where a caller is just doing a package create, they can use LoadPackageDefinitionWithValidate.